Gemfile

```diff
@@ -23,12 +23,13 @@ gem "stimulus-rails"
 
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem "jbuilder"
-gem "pry"  
+gem "pry"
```

Comment:
Move this gem to inside the developer and test environments.

Reply:
vale